code/API_definitions/Application_Endpoint_Discovery_API.yaml

```diff
@@ -0,0 +1,596 @@
+---
+openapi: 3.0.3
+info:
+  title: Application Endpoint Discovery API
+  version: 0.1.0-wip
+  description: |
+    # Connect to the Nearest Application Endpoint
+    ---
+    # Overview
+    The Application Discovery API extends beyond the capabilities of the
+    Simple Edge Discovery API by not only locating the nearest Edge Cloud Zone
+    but also directly linking to the application endpoints within those zones.
+    This crucial enhancement enables immediate connection to the most relevant
+    service endpoint, optimizing the application's interaction with
+    edge computing resources.
+    # Introduction
+    Operators often host multiple Edge Cloud Zones across various territories.
+    Connecting an application directly to the nearest operational service
+    endpoint minimizes latency because data travels the shortest possible route.
```

Comment:
On this "minimizes latency because data travels the shortest possible route", I think we may say "it may minimize latency" but we should not say "shortest possible route" as it is subjective and this API may not be providing any guarantee for this.

Reply:
Agree